smarts/core/smarts.py

```diff
@@ -1414,11 +1414,11 @@ def _try_emit_envision_state(self, provider_state: ProviderState, obs, scores):
                 if v.state == SignalLightState.OFF:
```

Comment:
Should this be changed too?
```suggestion
                if v.state == SignalLightState.Off:
```